modules/log-metric-filter/variables.tf

```diff
@@ -41,3 +41,9 @@ variable "metric_transformation_default_value" {
   type        = string
   default     = null
 }
+
+variable "metric_transformation_unit" {
+  description = "The unit to assign to the metric. If you omit this, the unit is set as None."
+  type        = string
+  default     = "None"
```

Comment:
Please update this default value to `null` and let Terraform AWS provider decide which value to use.

Reply:
@antonbabenko updated it. Thanks for the feedback